src/Simplex/Messaging/Crypto.hs

```diff
@@ -812,19 +812,19 @@ newtype Key = Key {unKey :: ByteString}
 -- | IV bytes newtype.
 newtype IV = IV {unIV :: ByteString}
 
--- | Key hash newtype.
-newtype KeyHash = KeyHash {unKeyHash :: ByteString} deriving (Eq, Ord, Show)
+-- | Certificate hash newtype.
+newtype CertificateHash = CertificateHash {unCertificateHash :: ByteString} deriving (Eq, Ord, Show)
```

Comment:
some yak shaving... let's please not rename in cases like this in the future.

Reply:
I actually think let's please rename back

Reply:
all that was needed to write in comment that it is actually a hash of the whole certificate, not just the key and keep the same, shorter type name